src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx

```diff
@@ -83,125 +73,103 @@ Session replay minimizes impact on your app's performance with:
     
 Set a sampling rate (0%-100%) for the following samples:
 
-    * <DoNotTranslate>**Session sampling rate**</DoNotTranslate> records a random sample of all user sessions.
+    * <DoNotTranslate>**User sessions**</DoNotTranslate> records a random sample of all user sessions.
 
-    * <DoNotTranslate>**Errored session sample rate**</DoNotTranslate> records a random sample of all errored sessions. The agent buffers for up to 30 seconds on the client-side, and if errors occur, the session will be recorded. If an error does not occur, the replay is not harvested. 
+    * <DoNotTranslate>**Errored session**</DoNotTranslate> records a random sample of all errored sessions. The agent buffers for up to 30 seconds on the client-side, and if errors occur, the session will be recorded. If an error does not occur, the replay is not harvested. 
 
-For example, if you set the session sampling rate to 50% and the error sampling rate to 100%, it means that:
+Here's an example of configuring sampling rates: If you set the session sampling rate to 50% and the error sampling rate to 100%, it means that:
 
 * Half of all user sessions will be recorded.
 * All errored sessions will be recorded, even if they're not part of the 50% sample.
 
   </Step>
+<Step>
+### Configure privacy settings [#configure-privacy-settings]
 
-    <Step>
-    ### Configure privacy settings [#configure-privacy-settings]
-
-    New Relic takes your data privacy seriously. The default setting masks everything by replacing all text with asterisks. 
-
-    Here's an example of how a replay looks with the default privacy settings:
-
-    <img
-        title="Example of default privacy settings"
-        alt="Example of default privacy settings"
-        src={browserTextMaskingExample}
-    />
-
-If you need to change the default privacy settings, you can choose specific classes and attributes to mask, block, or ignore:
-
-* <DoNotTranslate>**Mask**</DoNotTranslate> means specific text will be replaced with asterisks (*). For example, you could mask sensitive user input such as login information.
+<Callout variant="caution">
```

Comment:
I'm wondering if we can give more guidance/clarity here. Are we saying that we believe New Relic's default privacy settings are what people should be using and people should check with their privacy teams before changing them?